janusgraph-test/src/main/java/org/janusgraph/blueprints/AbstractJanusGraphProvider.java

```diff
@@ -134,7 +139,11 @@ public void clear(Graph g, final Configuration configuration) throws Exception {
             JanusGraph graph = (JanusGraph) g;
             if (graph.isOpen()) {
                 if (g.tx().isOpen()) g.tx().rollback();
-                g.close();
+                try {
+                    g.close();
+                } catch (IOException | IllegalStateException e) {
+                    logger.warn("Titan graph may not have closed cleanly", e);
```

Comment:
Understood thanks. I tried taking the try-catch out but I couldn't reproduce the issue in janusgraph-test or berkeleydb. So I would have had to run through the TinkerPop test suite in the cassandra and hbase modules to investigate further and each of those take 24 hours to run.